src/test/flag.test.ts

```diff
@@ -0,0 +1,19 @@
+import app from "app";
+import request from "supertest";
+import * as config from "../config";
+import { hasFeature } from "../utils/feature.flag";
+
+jest.mock("../utils/feature.flag");
+
+(hasFeature as jest.Mock).mockReturnValue({
+  "FLAG_1": false,
```

Comment:
Could have a pre-defined list of all the feature flags somewhere and then just override to set individual ones to true, in specific tests, when required.